Google.PowerShell/Logging/GcLogCmdlet.cs

```diff
@@ -359,4 +359,277 @@ protected override void ProcessRecord()
             while (!Stopping && logEntriesRequest.PageToken != null);
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates new monitored resources.
+    /// </para>
+    /// <para type="description">
+    /// Creates new monitored resources. These resources are used in the Logging cmdlets such as New-GcLogEntry
+    /// </para>
+    /// <example>
+    ///   <code>
+    ///   PS C:\> New-GcLogMonitoredResource -ResourceType gce_instance `
+    ///                                      -Labels @{"project_id" = "my-project"; "instance_id" = "my-instance"}.
+    ///   </code>
+    ///   <para>This command creates a new monitored resource of type gce_instance with specified labels.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/logging/docs/api/v2/resource-list)">
+    /// [Monitored Resources and Labels]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "GcLogMonitoredResource")]
+    public class NewGcLogMonitoredResource : GcLogCmdlet, IDynamicParameters
+    {
+        [Parameter(Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public Hashtable Labels { get; set; }
+
+        /// <summary>
+        /// This dynamic parameter dictionary is used by PowerShell to generate parameters dynamically.
+        /// </summary>
+        private RuntimeDefinedParameterDictionary dynamicParameters;
+
+        /// <summary>
+        /// This function is part of the IDynamicParameters interface.
+        /// PowerShell uses it to generate parameters dynamically.
+        /// </summary>
+        public object GetDynamicParameters()
```

Comment:
If this parameter always exists, why is this a dynamic parameter? A comment explaining would be useful.